backend/.env.dist

```diff
@@ -45,6 +45,7 @@ EMAIL_LINK_VERIFICATION=http://localhost/checkEmail/{optin}{code}
 EMAIL_LINK_SETPASSWORD=http://localhost/reset-password/{optin}
 EMAIL_LINK_FORGOTPASSWORD=http://localhost/forgot-password
 EMAIL_LINK_OVERVIEW=http://localhost/overview
+EMAIL_LINK_SUPPORT=https://gradido.net/de/contact/
```

Comment:
ich würde dies hier nicht fest in die .env setzen. der Support gilt ausschließlich für die Gradido Akademie. neu Communities werden einen anderen Support link haben. 
Im Text der E-Mail würde ich nur hinschreiben das sie sich an den Support wenden sollen. 
entweder ohne link oder im text den Link fest reinschreiben. hier an dieser stelle würde ich ihn nicht fest machen.

Reply:
Done 👍🏼